webpack/redux/actions/RedHatRepositories/helpers.js

```diff
@@ -27,6 +35,7 @@ const recommendedRepositoriesRHEL = [
 ];
 
 const recommendedRepositoriesSatTools = [
+  'satellite-client-6-for-rhel-10-x86_64-rpms',
```

Comment:
@ehelms do we know that the future Satellite client 6 repo for RHEL 10 will match this?

Reply:
Yes - it should, as far as we know, follow the same pattern as RHEL 9.